serverLib.sml

```diff
@@ -456,40 +466,50 @@ structure ReadJSON = struct
       val () = if Substring.isEmpty e then () else raise Option
     in (date, ss) end
 
+  fun skip_space ss =
```

Comment:
Why is all this space skipping suddenly necessary?

Reply:
I see in your commit message that it's because you haven't easily been able to produce compacted responses from GitHub. I think GitHub sends compacted responses when you query its API... what other method are you using to generate responses that doesn't?

Reply:
Well, this one: https://developer.github.com/v4/explorer/

Note there's two commits here. If you want to ignore the space-handling, just read the diff of the first one.

Reply:
How about this: replace the space-handling commit with a smaller commit that strips all spaces from the response in one place only, and possibly comment out that lone call to the stripping function (because I don't think it's needed in production). Would that still be usable for debugging with the explorer?

Reply:
Not much point keeping the code around, you might as well pass the responses through
tr -d "[:space:]".